packages/astro/src/webcontainer-files/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const IGNORED_FILES = ['**/.DS_Store', '**/*.swp'];
+export const EXTEND_CONFIG_FILEPATH = '/.tk-config.json';
```

Comment:
Interested to hear feedback / thoughts on the name of the config file here. :eyes: 

I started with `template.json`, then renamed it to `.template.json`, then `.config.json` to finally going with `.tk-config.json`.

My reasoning has been: I started with `template.json` because this is for templates. Then I thought, this might collide with user files, so added the starting `.`. I then realised that the code would also work for `_files` and `_solution` because we don't distinguish them and so thought maybe a more generic name would be better. I went with `.config.json` but figured that adding a `tk-` prefix would make it more explicitely TutorialKit specific and help tooling like the VSCode extension, to provide a schema for the JSON.

cc @sulco